Extension/src/LanguageServer/client.ts

```diff
@@ -2437,6 +2439,15 @@ export class DefaultClient implements Client {
         });
     }
 
+    /**
+     * notifications to the language server
```

Comment:
"notifications to the language server" is generic as there are other functions that sends notifications to the language server. Suggest to change to something specific that describes what `restartLanguageServerActiveFile` is uniquely doing.